.github/workflows/learn-github-actions.yml

```diff
@@ -0,0 +1,13 @@
+name: learn-github-actions
```

Comment:
- Optional
- The name of the workflow
  - as it will appear in the "Actions" tab of the GitHub repository
- If this field is omitted, 
  - the name of the workflow file will be used instead.